packages/release-brancher/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "release-brancher",
+  "version": "1.0.0",
+  "description": "Cut release branches",
+  "bin": "./build/src/bin/release-brancher.js",
```

Comment:
Do we need to worry about any auto-release or update scripts happening, like they do in other bot directories? Or is that all localized in `cloudbuild.yaml` files now?

Reply:
Can you elaborate on this? I added this b/c I though it was how it would link a binary when you installed this library.

Reply:
Sorry, my comment was in an unfortunately random spot.  This was a general concern - there used to be a script that iterated over every directory in `packages`, and tried to publish it to a cloud function.  Is that still happening?

Reply:
That script looks for `cloudbuild.yaml` files and sets up triggers if it finds one. We intentionally do not include one here.